src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * 
+ */
+package seedu.addressbook.data.person;
+
+/**
+ * @author Bel
+ * Represents a person's block in the address of that person.
```

Comment:
like I pointed out in the tutorial, the order of these 2 lines should be swapped